poorwsgi/response.py

```diff
@@ -420,6 +406,41 @@ def __init__(self, arg: Union[int, Response], **kwargs):
         assert isinstance(arg, (int, Response))
         super().__init__(arg, kwargs)
 
+    def make_response(self):
+        """Return or make response if is possible."""
+        if isinstance(self.args[0], Response):
+            return self.args[0]
+
+        status_code = self.args[0]
+        if status_code == DECLINED:
+            return Decline()   # decline the connection
+        if status_code == HTTP_OK:
+            return EmptyResponse()
+        return None
+
+
+def make_response(data: Union[str, bytes],
+                  content_type: str = "text/html; charset=utf-8",
+                  headers: Union[Headers, HeadersList] = None,
+                  status_code: int = HTTP_OK):
+    """Create response from simple values.
```

Comment:
What do you want to say with `simple`? I'd use `simple` in regard with an algorithm, for example. If you want to say that the values are not composite structures (objects), I'd use `primitive` or `scalar` values.